lib/stdlib/src/shell.erl

```diff
@@ -164,7 +164,7 @@ server(StartSync) ->
     RT = ets:new(?RECORDS, [public,ordered_set]),
     _ = initiate_records(Bs, RT),
     process_flag(trap_exit, true),
-    %% Store function definitions and types in an ets table.
+    %% Store local definitions of functions, records and types in an ets table.
     FT = ets:new(user_functions, [public,ordered_set]),
```

Comment:
Would you mind writing a comment describing what type of elements are places inside the ets table and what they mean? Especially the differentce between the `_def` and non-`_def` variants.